frontend/src/components/abfragevarianten/ReportsPlanungsursaechlichkeitComponent.vue

```diff
@@ -49,6 +49,26 @@
             </a>
           </v-list-item-title>
         </v-list-item>
+        <v-list-item>
+          <v-list-item-title>
+            <a
+              target="_blank"
+              :href="getUrlPlanungsursaechlicheSpitzenbedarfeKinderkrippe()"
+            >
+              Report planungsursaechliche Spitzenbedarfe Kinderkrippe<span class="mdi mdi-launch" />
+            </a>
+          </v-list-item-title>
+        </v-list-item>
+        <v-list-item>
+          <v-list-item-title>
+            <a
+              target="_blank"
+              :href="getUrlPlanungsursaechlicheSpitzenbedarfeKindergarten()"
+            >
+              Report planungsursaechliche Spitzenbedarfe Kindergarten<span class="mdi mdi-launch" />
```

Comment:
```suggestion
              Report planungsursächliche Spitzenbedarfe Kindergarten<span class="mdi mdi-launch" />
```

Umlaute dürfen vorkommen.